.ci/pull-request-config.yaml

```diff
@@ -3,7 +3,7 @@ version: "2.1"
 dependencies: ./project-dependencies.yaml
 
 build:
-  - project: apache/incubator-kie-drools
+  - project: kiegroup/drools
```

Comment:
I noticed that the previous `apache/incubator-kie-drools` was incompatible with `kiegroup/drools`. If we go with `kiegroup/drools`, this line should also align.